pkg/e2e/operators/splunkforwarder.go

```diff
@@ -39,4 +47,73 @@ var _ = ginkgo.Describe(splunkForwarderBlocking, func() {
 	checkUpgrade(helper.New(), "openshift-splunk-forwarder-operator",
 		"openshift-splunk-forwarder-operator", "splunk-forwarder-operator",
 		"splunk-forwarder-operator-catalog")
+
+	ginkgo.Context("splunkforwarders", func() {
+		ginkgo.It("dedicated admin should not be able to manage splunkforwarders CR", func() {
+			sf := makeMinimalSplunkforwarder("SplunkForwarder", "splunkforwarder.managed.openshift.io/v1alpha1", "osde2e-splunkforwarder-test-1")
+			err := dedicatedAaddSplunkforwarder(sf, h)
```

Comment:
We should defer a cleanup on both of these: https://github.com/openshift/osde2e/blob/00cde4de6721e1b4c182afe85860e6eae8332eee/pkg/e2e/verify/user_webhook.go#L51-L54

Reply:
Thanks for reviewing @cblecker! I updated the code accordingly.